README.md

```diff
@@ -1,12 +1,14 @@
-# autoreduce
-ISIS Autoreduction service.
-An open source service designed to automatically reduce instrument data from the ISIS pulsed neutron and muon source. 
-
----
-## Status
-### Master
-[![Build Status](https://travis-ci.org/ISISScientificComputing/autoreduce.svg?branch=master)](https://travis-ci.org/ISISScientificComputing/autoreduce)
-[![Master Coverage Status](https://coveralls.io/repos/github/ISISScientificComputing/autoreduce/badge.svg?branch=master)](https://coveralls.io/github/ISISScientificComputing/autoreduce?branch=master)
-### Develop
-[![Dev Build Status](https://travis-ci.org/ISISScientificComputing/autoreduce.svg?branch=develop)](https://travis-ci.org/ISISScientificComputing/autoreduce)
-[![Develop Coverage Status](https://coveralls.io/repos/github/ISISScientificComputing/autoreduce/badge.svg?branch=develop)](https://coveralls.io/github/ISISScientificComputing/autoreduce?branch=develop)
+| Master        | Develop       |
+| :-------------|:--------------|
+| [![Build Status](https://travis-ci.org/ISISScientificComputing/autoreduce.svg?branch=master)](https://travis-ci.org/ISISScientificComputing/autoreduce) [![Master Coverage Status](https://coveralls.io/repos/github/ISISScientificComputing/autoreduce/badge.svg?branch=master)](https://coveralls.io/github/ISISScientificComputing/autoreduce?branch=master) | [![Dev Build Status](https://travis-ci.org/ISISScientificComputing/autoreduce.svg?branch=develop)](https://travis-ci.org/ISISScientificComputing/autoreduce) [![Develop Coverage Status](https://coveralls.io/repos/github/ISISScientificComputing/autoreduce/badge.svg?branch=develop)](https://coveralls.io/github/ISISScientificComputing/autoreduce?branch=develop) |
+
+# Autoreduction
+The software and setup for a service designed to automatically reduce instrument data. Used at [ISIS Neutron and Muon Facility](https://www.isis.stfc.ac.uk). For further documentation see also [Wiki](https://github.com/ISISScientificComputing/autoreduce/wiki). 
+
+In one (not complete) picture the Autoreduction service is:
+
+![Example Table](Documentation/docs_for_main_readme/Autoreduction_main_components.png)
```

Comment:
This diagram is definitely good to have, however can we remove the red underline from the word WebApp and then re-upload it?